src/pages/home/report/ReportActionsView.js

```diff
@@ -414,7 +414,7 @@ ReportActionsView.defaultProps = defaultProps;
 
 export default compose(
     withWindowDimensions,
-    withDrawerState,
+    // withDrawerState,
```

Comment:
Getting the same exception here.

Reply:
https://github.com/Expensify/Expensify.cash/blob/1292b83b26d2001f4e2fd34b1743a5b08f39f379/src/components/withDrawerState.js#L12

Switch to `useDrawerStatus()`
```
 const drawerStatus = useDrawerStatus();
```


https://github.com/Expensify/Expensify.cash/blob/1292b83b26d2001f4e2fd34b1743a5b08f39f379/src/components/withDrawerState.js#L19
```
  isDrawerOpen={drawerStatus === 'open'}
  
```  

Hope it helps. 😃

Reply:
Yes, it resolved the exceptions. Thanks!